src/Sylius/Bundle/ProductBundle/Form/Type/ProductType.php

```diff
@@ -30,7 +30,7 @@ class ProductType extends AbstractResourceType
     private $variantResolver;
 
     /**
-     * @param string $dataClass FQCN
+     * @param string $dataClass
```

Comment:
Then we should add it in every form type with defined/overwritten constructor :) In my opinion it's quite obvious and most of all we should be consistent with this in all form types.